posts/2021-08-31-open-liberty-starter.adoc

```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+title: "Get started using the Open Liberty Starter"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/steven1046
+author_github: https://github.com/steven1046
+seo-title: Yet another way to get started with Open Liberty - OpenLiberty.io
+seo-description: Yet another way to get started with Open Liberty
+blog_description: Yet another way to get started with Open Liberty
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= Open Liberty Starter
+Steven Zvonek <https://github.com/steven1046>
+:imagesdir: /
+:url-prefix: https://openliberty.io
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+[#Intro]
+== Open Liberty Starter Application
+
+There is now another easy way to get started using Open Liberty. Visit the link:{url-prefix}/downloads/[Getting Started] page and select a few options to generate a starter application using Open Liberty. You can specify your application and project name, choose a build tool from either Maven or Gradle, and pick which version of Java SE, Jakarta EE, and MicroProfile your application will use. Then, just click `Generate Project` and you are ready for lift-off! 
```

Comment:
I think we should add another paragraph that explains what to do once they download the package (unzip & what command(s) to run).  (Really, I think this should be included with the starter, on the site and/or a README in the zip.  
Lastly, maybe provide links to  (https://openliberty.io/guides/?search=microprofile&key=tag) and Jakarta EE guides (https://openliberty.io/guides/?search=jakarta%20ee&key=tag) for further info on those.

Reply:
We do have https://github.com/OpenLiberty/start.openliberty.io/issues/9, so i could add some of the commands in there and briefly talk about it in the blog.

Reply:
I have no issues with us adding extra instructions to the README.md of start.openliberty.io repo, but that should not be the main way that users obtain that information nor should that be used in the blog.  For the blog, just directly provide the information (which you can then also paste into the readme.md of that repo.

Long-term, we should update the user flow to provide them with this information.  For example, once the user clicks "Generate Projects" on /downloads, we could then show them the commands based on which build project they selected (and we could also include those commands in some sort of file in the zip, again, pertaining to the build system they selected).  I've added this to the agenda for our ol.io call.